### PR TITLE
Anomali ThreatStream DBot Score support

### PR DESCRIPTION
**Pull request by Itay4**
Fixes https://github.com/demisto/etc/issues/10206

### CODE REVIEW

**Comment by anara123** [comment]
@meirwah can you review it?

File: Integrations/integration-AnomaliThreatStream.yml [hunk 1]
```diff
@@ -243,6 +243,20 @@ script:
...
+        var dbotScore = 1;
+        if (output['Threat Score'] >= args.threshold) {
+            dbotScore = 3;
+        } else if (output['Threat Score'] > 0) {
+            dbotScore = 2;
+        }
+
+        entry.EntryContext.DBotScore = {
+                                        Indicator: args.value,
+                                        Type: args.type,
```
Comment (meirwah) [inline Integrations/integration-AnomaliThreatStream.yml]: args.type ?
Those should be consts ...

File: Integrations/integration-AnomaliThreatStream.yml [hunk 2]
```diff
@@ -243,6 +243,20 @@ script:
...
+                                        Vendor: 'ThreatStream',
```
Comment (anara123) [inline Integrations/integration-AnomaliThreatStream.yml]: fix the indentation

File: Integrations/integration-AnomaliThreatStream.yml [hunk 3]
```diff
@@ -243,6 +243,20 @@ script:
             entry.EntryContext[contextKey] = contexts;
         }
 
+        var dbotScore = 1;
+        if (output['Threat Score'] >= args.threshold) {
```
[the remark this answers is not in the document]
Comment (Itay4) [reply Integrations/integration-AnomaliThreatStream.yml]: According to their documentation, 'Threat Score' will always be returned from every query.

File: Integrations/integration-AnomaliThreatStream.yml (hunk 1 above)
Comment (Itay4) [reply Integrations/integration-AnomaliThreatStream.yml]: This is not really an argument. If you would look at the switch-case, you would find out that each 'type' of command is set into 'args'.

**Pull request merged**